server/app/models/Product.js

```diff
@@ -30,7 +30,7 @@ const ProductSchema = mongoose.Schema(
     lists: {
       type: [
         {
-          type: mongoose.Schema.ObjectId,
+          type: mongoose.Schema.Types.ObjectId,
```

Comment:
I did some research into what the difference is between ` mongoose.Schema.ObjectId` and `mongoose.Schema.Types.ObjectId` and this [stackflow link](https://stackoverflow.com/questions/28997636/should-i-use-schema-types-objectid-or-schema-objectid-when-defining-a-mongoose-s) shows that they're the same.